schema/bls.mcf

```diff
@@ -15329,3 +15329,43 @@ name: "State and local government education"
 Node: dcid:NAICS/JOLTS_929000
 typeOf: dcs:NAICSEnum
 name: "State and local government excluding education"
+
+Node: dcid: consumerPriceIndex
+typeOf: schema:property
+domainIncludes: dcs:ConsumerGoodsAndServices
+rangeIncludes: schema:Number
+description: "A Consumer Price Index measures changes in the price level of a weighted average market basket of consumer goods and services purchased by households."
+descriptionUrl: "https://en.wikipedia.org/wiki/Consumer_price_index"
+
+Node: dcid:IndexPointBasePeriod1982-84=100
+name: "IndexPointBasePeriod1982-84=100"
+typeOf: dcs:UnitOfMeasure
+description: "The reference base is 1982 to 1984 equals 100."
+descriptionUrl: "https://www.bls.gov/cpi/technical-notes/home.htm"
+
+Node: dcid:IndexPointBasePeriod1999-12=100
+name: "IndexPointBasePeriod1999-12=100"
+typeOf: dcs:UnitOfMeasure
+description: "The reference base is December 1999 equals 100."
+descriptionUrl: "https://www.bls.gov/cpi/technical-notes/home.htm"
```

Comment:
Should we expand out the symbols I wonder:
- IndexPointBasePeriod1982To84Equals100
- IndexPointBasePeriodDec1982Equals100

Reply:
I am down but I need a confirmation

Reply:
I think that sounds like a good idea.

Reply:
changed